packages/editor/src/lib/exports/html-to-image/fetchCache.ts

```diff
@@ -0,0 +1,27 @@
+import { FileHelpers, assert, fetch } from '@tldraw/utils'
+
+export function fetchCache<T>(cb: (response: Response) => Promise<T>, init?: RequestInit) {
+	const cache = new Map<string, Promise<T | null>>()
```

Comment:
y e s... i'm not sure what makes the most sense here.

On the one hand, it does make things _much_ faster. There are certain things I think we should sort of just always cache, like fonts. We could maybe do some sort of LRU cache for other things tho... I'll have a bit of a think about what we can do here

Reply:
it's funny because i can hear you actually saying 'y e s` in my head :P

no worries for this PR, I think for all intents and purposes it's probably fine. but then again, for bigger canvases, it might add up.